### DISCUSSION TITLE
sourceMappingURL in dist css files since 0.15

### DISCUSSION THREAD

**Discussion by krzyko** [selectize/selectize.js#1913] [Ideas]
Since 0.15 version all css files located in dist folder include sourceMappingURL which makes them about three times heavier.

**Comment by risadams** [comment]
This was intentional, but I get the concerns about size.
What is the general consensus from the community at large?

**Reply by krzyko** [reply]
I used to have "filename.css" for unminified version to simplify debugging on development server, and minified "filename.min.css" for fastest processing on release server.
Here "selectize.css" (and other stylesheets) are minified with sourceMappingURL which, for me, is also OK for development purposes, but for release server I'd like to have these files with sourceMappingURL stripped. Of course I can cut files by myself, but then it isn't possible to get them from public CDN server (like jsdelivr).
Adding minified stylesheets with sourceMappingURL stripped seems to be the simpliest solution.

**Reply by risadams** [reply]
This is a fair enough request.  I'm pulling the source maps out, and if anyone does need them, they can do a custom build from the sass/less anyway.

Pushing a v0.15.2 release out now
